heron/ui/src/python/version.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
putting them into ``utils`` module under ``heron/common`` looks strange to me. I can remove these ``version.py`` and put this simple logic of adding parser inside ``main.py``.